WordPress/Classes/ViewRelated/Blog/Blog Details/BlogDetailsViewController.m

```diff
@@ -1670,10 +1670,10 @@ - (void)showBlaze
 {
     [BlazeEventsTracker trackEntryPointTappedFor:BlazeSourceMenuItem];
     
-    [BlazeOverlayCoordinator presentBlazeOverlayInViewController:self
-                                                          source:BlazeSourceMenuItem
-                                                            blog:self.blog
-                                                            post:nil];
+    [BlazeFlowCoordinator presentBlazeInViewController:self
+                                                        source:BlazeSourceMenuItem
+                                                        blog:self.blog
+                                                        post:nil];
```

Comment:
[nit] alignment

Reply:
Fixed in 885a79e62e062dd8d7699022bbea5d65aefd004d 👍